pkg/reconciler/v1alpha1/revision/resources/queue.go

```diff
@@ -135,6 +136,9 @@ func makeQueueContainer(rev *v1alpha1.Revision, loggingConfig *logging.Config, a
 		}, {
 			Name:  "USER_PORT",
 			Value: strconv.Itoa(int(userPort)),
+		}, {
+			Name:  system.NamespaceEnvKey,
```

Comment:
I think this requires an update to `queue_test.go`, which will need to link our new library.